checkstyle-custom/src/test/resources/io/vivy/checkstyle/custom/TestMethodCallParenMatchCheckErrors.java

```diff
@@ -20,9 +20,17 @@ public void testMethod() {
 
     public static String doNothing(String nothing) {
 
-        return new StringBuffer(
-           nothing.toUpperCase()
-        );
+        return nothing.concat(
+           "Nothing"
+               );
```

Comment:
`);` should match `return`

Reply:
Yeah but that was part of the issue with the second issue I mentioned in the git as return is directly not part of the method call tokens  http://checkstyle.sourceforge.net/apidocs/com/puppycrawl/tools/checkstyle/api/TokenTypes.html#METHOD_CALL

Reply:
updated